<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>670</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>880</width>
    <height>670</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>880</width>
    <height>670</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>811</width>
      <height>571</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <widget class="QFrame" name="frameX">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>200</width>
       <height>530</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="buttonDisconnectX">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>465</y>
        <width>120</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string>Disconnect</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonSetZeroX">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>185</y>
        <width>120</width>
        <height>35</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Set Zero</string>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonConnectX">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>465</y>
        <width>120</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditSetVX">
      <property name="geometry">
       <rect>
        <x>45</x>
        <y>150</y>
        <width>110</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_setJ_x">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>270</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Jog Step</string>
      </property>
     </widget>
     <widget class="QLabel" name="currentVoltage_5">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>20</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>V</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonDisableX">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>420</y>
        <width>120</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string>Disable</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_setV_x">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>70</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Voltage</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonEnableX">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>420</y>
        <width>120</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string>Enable</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_x_direction">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>x direction</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumberCVX">
      <property name="geometry">
       <rect>
        <x>45</x>
        <y>110</y>
        <width>110</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="smallDecimalPoint">
       <bool>false</bool>
      </property>
      <property name="mode">
       <enum>QLCDNumber::Dec</enum>
      </property>
     </widget>
     <widget class="QLabel" name="currentVoltage_3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>290</y>
        <width>20</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>V</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumberCJX">
      <property name="geometry">
       <rect>
        <x>45</x>
        <y>300</y>
        <width>110</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="smallDecimalPoint">
       <bool>false</bool>
      </property>
      <property name="mode">
       <enum>QLCDNumber::Dec</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditSetJX">
      <property name="geometry">
       <rect>
        <x>45</x>
        <y>340</y>
        <width>110</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frameY">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>200</width>
       <height>530</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="buttonConnectY">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>465</y>
        <width>120</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditSetVY">
      <property name="geometry">
       <rect>
        <x>45</x>
        <y>150</y>
        <width>110</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonDisableY">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>420</y>
        <width>120</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string>Disable</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonSetZeroY">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>185</y>
        <width>120</width>
        <height>35</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Set Zero</string>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumberCJY">
      <property name="geometry">
       <rect>
        <x>45</x>
        <y>300</y>
        <width>110</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="smallDecimalPoint">
       <bool>false</bool>
      </property>
      <property name="mode">
       <enum>QLCDNumber::Dec</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonDisconnectY">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>465</y>
        <width>120</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string>Disconnect</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditSetJY">
      <property name="geometry">
       <rect>
        <x>45</x>
        <y>340</y>
        <width>110</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_setJ_Y">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>270</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Jog Step</string>
      </property>
     </widget>
     <widget class="QLabel" name="currentVoltage_4">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>290</y>
        <width>20</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>V</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_setV_y">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>70</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Voltage</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_y_direction">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>y direction</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonEnableY">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>420</y>
        <width>120</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string>Enable</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumberCVY">
      <property name="geometry">
       <rect>
        <x>45</x>
        <y>110</y>
        <width>110</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="smallDecimalPoint">
       <bool>false</bool>
      </property>
      <property name="mode">
       <enum>QLCDNumber::Dec</enum>
      </property>
     </widget>
     <widget class="QLabel" name="currentVoltage_6">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>20</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>V</string>
      </property>
     </widget>
     <zorder>lineEditSetVY</zorder>
     <zorder>buttonDisableY</zorder>
     <zorder>buttonSetZeroY</zorder>
     <zorder>lcdNumberCJY</zorder>
     <zorder>buttonDisconnectY</zorder>
     <zorder>lineEditSetJY</zorder>
     <zorder>label_setJ_Y</zorder>
     <zorder>currentVoltage_4</zorder>
     <zorder>label_setV_y</zorder>
     <zorder>label_y_direction</zorder>
     <zorder>buttonEnableY</zorder>
     <zorder>lcdNumberCVY</zorder>
     <zorder>currentVoltage_6</zorder>
     <zorder>buttonConnectY</zorder>
    </widget>
    <widget class="QFrame" name="frameJogging">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>20</y>
       <width>350</width>
       <height>421</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="buttonMoveLeft">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>130</y>
        <width>100</width>
        <height>70</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonMoveDown">
      <property name="geometry">
       <rect>
        <x>125</x>
        <y>200</y>
        <width>100</width>
        <height>70</height>
       </rect>
      </property>
      <property name="text">
       <string>v</string>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonMoveUp">
      <property name="geometry">
       <rect>
        <x>125</x>
        <y>60</y>
        <width>100</width>
        <height>70</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>^</string>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonMoveRight">
      <property name="geometry">
       <rect>
        <x>225</x>
        <y>130</y>
        <width>100</width>
        <height>70</height>
       </rect>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_jog_voltage">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Jog Voltage</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonSwitchX">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>150</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string>Switch x direction</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonSwitchY">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>370</y>
        <width>150</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string>Switch y direction</string>
      </property>
     </widget>
    </widget>
    <zorder>frameJogging</zorder>
    <zorder>frameY</zorder>
    <zorder>frameX</zorder>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>880</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionChangeSerial"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionChangeSerial">
   <property name="text">
    <string>Change Piezo Serial #</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
